native/cocos/renderer/pipeline/GeometryRenderer.cpp

```diff
@@ -810,6 +811,25 @@ void GeometryRenderer::addBezier(const Vec3 &v0, const Vec3 &v1, const Vec3 &v2,
     }
 }
 
+void GeometryRenderer::addSpline(const geometry::Spline &spline, gfx::Color color, uint32_t index, float knotSize, uint32_t segments, bool depthTest) {
+    const auto numPoints = segments + 1;
+    auto       points    = spline.getPoints(numPoints, index);
+
+    for (auto i = 0U; i < segments; i++) {
+        addLine(points[i], points[i + 1], color, depthTest);
+    }
+
+    if (knotSize > 0.0F && index == 0xffffffff) {
```

Comment:
It is better to use SPLINE_WHOLE_INDEX instead of 0xffffffff here.

Reply:
There is some problem when exporting this value in jsb

Reply:
It is in a cpp file, how can it affect jsb?

Reply:
1